### PR TITLE
Reword medal unlock requirements index page

### PR DESCRIPTION
**Pull request by cl8n**
the hush-hush note sounded kind of condescending for a wiki :thinking:

### CODE REVIEW

File: wiki/Medals/Unlock_requirements/ru.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+---
```
Comment (minusQuantumNeko) [inline wiki/Medals/Unlock_requirements/ru.md:1]:
```suggestion
```

File: wiki/Medals/Unlock_requirements/ru.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+outdated_since: 8951e9fef0bc292f2b085b7c3a400db936392e49
```
Comment (minusQuantumNeko) [inline wiki/Medals/Unlock_requirements/ru.md:2]:
```suggestion
```

File: wiki/Medals/Unlock_requirements/ru.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
...
+outdated_translation: true
```
Comment (minusQuantumNeko) [inline wiki/Medals/Unlock_requirements/ru.md:3]:
```suggestion
```

File: wiki/Medals/Unlock_requirements/ru.md [hunk 4]
```diff
@@ -1,3 +1,8 @@
...
+---
```
Comment (minusQuantumNeko) [inline wiki/Medals/Unlock_requirements/ru.md:4]:
```suggestion
```

File: wiki/Medals/Unlock_requirements/ru.md [hunk 5]
```diff
@@ -1,3 +1,8 @@
...
+
```
Comment (minusQuantumNeko) [inline wiki/Medals/Unlock_requirements/ru.md:5]:
```suggestion
```

File: wiki/Medals/Unlock_requirements/ru.md [hunk 6]
```diff
@@ -1,3 +1,8 @@
...
 # Условия получения медалей
 
 Индексная страница, где перечислены все условия получения медалей. Медали категории Hush-Hush по понятным причинам сюда не входят.
```
Comment (minusQuantumNeko) [inline wiki/Medals/Unlock_requirements/ru.md:8]:
```suggestion
Индексная страница, где перечислены статьи с условиями получения [медалей](/wiki/Medals).

Условия для медалей категории [Hush-Hush](/wiki/Medals#hush-hush) не перечислены на вики, чтобы помочь сохранить их секретность.
```

File: wiki/Medals/Unlock_requirements/uk.md [hunk 7]
```diff
@@ -1,3 +1,8 @@
+---
```
Comment (minusQuantumNeko) [inline wiki/Medals/Unlock_requirements/uk.md:1]:
```suggestion
```

File: wiki/Medals/Unlock_requirements/uk.md [hunk 8]
```diff
@@ -1,3 +1,8 @@
...
+outdated_since: 8951e9fef0bc292f2b085b7c3a400db936392e49
```
Comment (minusQuantumNeko) [inline wiki/Medals/Unlock_requirements/uk.md:2]:
```suggestion
```

File: wiki/Medals/Unlock_requirements/uk.md [hunk 9]
```diff
@@ -1,3 +1,8 @@
...
+outdated_translation: true
```
Comment (minusQuantumNeko) [inline wiki/Medals/Unlock_requirements/uk.md:3]:
```suggestion
```

File: wiki/Medals/Unlock_requirements/uk.md [hunk 10]
```diff
@@ -1,3 +1,8 @@
...
+---
```
Comment (minusQuantumNeko) [inline wiki/Medals/Unlock_requirements/uk.md:4]:
```suggestion
```

File: wiki/Medals/Unlock_requirements/uk.md [hunk 11]
```diff
@@ -1,3 +1,8 @@
...
+
```
Comment (minusQuantumNeko) [inline wiki/Medals/Unlock_requirements/uk.md:5]:
```suggestion
```

File: wiki/Medals/Unlock_requirements/uk.md [hunk 12]
```diff
@@ -1,3 +1,8 @@
...
 # Вимоги для отримання
 
 Індексна сторінка для всіх статей відносно вимог для отримання медалей. Hush-Hush медалі не включені зі зрозумілих причин.
```
Comment (minusQuantumNeko) [inline wiki/Medals/Unlock_requirements/uk.md:8]:
```suggestion
Індексна сторінка для всіх статей відносно вимог для отримання [медалей](/wiki/Medals).

Вимоги для категорії медалей [Hush-Hush](/wiki/Medals#hush-hush) не включені сюди, щоб допомогти зберегти їх секретність.
```

**Review by minusQuantumNeko — CHANGES_REQUESTED** [review]

**Comment by cl8n** [comment]
open new pr for translations if/when this one is merged

**Comment by minusQuantumNeko** [comment]
:<

**Review by minusQuantumNeko — APPROVED** [review]

**Review by Hiviexd — APPROVED** [review]

**Pull request merged**